### ISSUE TITLE
Once or double ^ or _ does {} if math end environments command is after ^ or _

### ISSUE THREAD

**Issue by AntRay** [randy3k/LaTeXYZ#18]
When I write in math mode, for example $x^2$ or $x_2$, it will produce $x{2}$. When I write $x^2 $, it works correctly. The issue also works for two ^ or _. $x^^2$ or $x__2$ will produce $x{{2}}$. And when I write this with a space at the end: $x^^2 $ will produce $x^{2} $.

Can you help me to resolve this issue?

**Comment by BigBIgMouse** [comment]
Oh man I have the same issue and I'm messed up by this.

**Comment by AntRay** [comment]
> Have you solved it?

Yes, I did. I do not know what you need to do. I just deleted all of my LaTeX packages in ST3 and reinstalled or leaved untouched LaTeXTools and LaTeXYZ packages (for speedily and comfortable work it are just enough). After these actions it works correctly.

> I have no idea why it occurs. What packages have you installed? Maybe there is a crash with other packages?

Yes it is. The issue occurs when you have an another LaTeX package installed except LaTeXTools and LaTeXYZ
